### ISSUE TITLE
Automatic update of KISAO on obo foundry and ontology lookup service (OLS)

### ISSUE THREAD

**Issue by matthiaskoenig** [SED-ML/KiSAO#103]
Currently the ontology lookup service (OLS) and opo foundry still use old versions of KISAO.
This should be fixed as soon as the following is merged:
https://github.com/OBOFoundry/purl.obolibrary.org/issues/787
Just opening this here to keep track of this and keep people in the loop.

Best Matthias

**Comment by jonrkarr** [comment]
I've been trying to get this updated. The first link in the chain of sources has been updated for the migration to github. This still hasn't propagated through to OLS. I've talked to EBI about it. They're waiting on updates from their source. I think this is stuck in something that has an infrequent release schedule.

**Comment by matthiaskoenig** [comment]
As I understood it they rely on obofoundry. As soon as this is updated OLS should follow with their next release.

**Comment by jonrkarr** [comment]
Correct. Unfortunately, the release schedule is slow. The record in OLS was updated months ago.

**Comment by jonrkarr** [comment]
BioPortal is up to date

**Comment by jonrkarr** [comment]
The KiSAO record in OBO foundry has been updated. See below.

https://github.com/OBOFoundry/OBOFoundry.github.io/blob/master/ontology/kisao.md

**Comment by matentzn** [comment]
(Just for the record, the problem was the missing purl config, not metadata, that @matthiaskoenig added here: https://github.com/OBOFoundry/purl.obolibrary.org/pull/792/files) 

Once https://github.com/OBOFoundry/purl.obolibrary.org/pull/792 is merged, you should ask the OLS team to prioritise reloading KISAO and from that point on, it will be fine.
